bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToZATo2L2Tau/run2Template_cards/template_AToZHTo2L2Tau_200_50_1_ggH_TuneCP5_13TeV_pythia8/AToZHTo2L2B_200_50_1_ggH_TuneCP5_13TeV_pythia8_customizecards.dat

```diff
@@ -0,0 +1,15 @@
+# Settings for the Higgs potential 
+set param_card higgs 1 0.0655307646
+set param_card higgs 2 0.0655307646
+set param_card higgs 3 0.161002788
+# Settings 2hdm parameters 
+set param_card frblock 1 1.0     # tanbeta 
+set param_card frblock 2 0.99    # sinbma 
+# Settings for the scalar bosons masses 
+set param_card mass 25 125.0     # h1: h
+set param_card mass 35 200.0     # h2: H
+set param_card mass 36 50.0      # h3: A
+set param_card mass 37 200.0     # hc: Hc
+# Settings for the scalar widths 
+set param_card width 36 Auto
+set param_card width 35 0.96446288
```

Comment:
How are the widths set? In your py script I see that you calculate widths for H and A, but then why is it auto and a fixed value?